src/main/java/no/nav/fo/veilarbdirigent/handlers/DialogHandler.java

```diff
@@ -45,7 +79,13 @@ public void register() {
                         new Task<>()
                                 .withId(String.valueOf(msg.getId()) + "dialog")
                                 .withType(TYPE)
-                                .withData(new TypedField<>(new OppfolgingData(msg))));
+                                .withData(new TypedField<>(new OppfolgingData(msg, sykemeldtDialog))));
+            } else if (behovForArbeidsevnevurdering.equals(msg.getForeslattInnsatsgruppe())) {
+                return List.of(
+                        new Task<>()
+                                .withId(String.valueOf(msg.getId()) + "dialog")
```

Comment:
Vil helst ha mer unike id-er her. for å enklere diffe den i databasen i etterkant

Reply:
sykemeldtDialog og arbeidsevneDialog i stedet